feat: new layout

rodrigopaixaorj opened the pull request:
For more informations about what we've done, please visit: https://github.com/Nottinghster/otclient/tree/new-layout

rodrigopaixaorj committed:
New Modules

rodrigopaixaorj committed:
New Data

kokekanon committed:
improvement: game_entergame

rodrigopaixaorj committed:
improvement: [BETA] game_entergame

kokekanon committed:
format

kokekanon committed:
last version game entergame

kokekanon committed:
Universal

rodrigopaixaorj committed:
Merge pull request #2 from kokekanon/new-layout

New layout

rodrigopaixaorj committed:
fix: added game_stash and other things

* Added Game Stash (removed accidentaly
* Fixed error when open outfit window
* Fixed health/mana bars order in right panel
 * Other minor fixes

kokekanon committed:
fix: game_console WASD and add missing funcion bot

fix: game_console WASD and add missing funcion bot

rodrigopaixaorj committed:
Merge pull request #3 from kokekanon/new-layout

fix: game_console WASD and add missing funcion bot

kokekanon committed:
Fix Cap | soul | format

rodrigopaixaorj committed:
Merge branch 'new-layout' into new-layout

rodrigopaixaorj committed:
Merge pull request #4 from kokekanon/new-layout

Fix: Cap | soul | format

rodrigopaixaorj committed:
fix: minor fixes

kokekanon committed:
feat: Game Skills of tibia

kokekanon committed:
improve : change of location shader " OTCLient Redemption"

kokekanon committed:
fix: old protocol hide unnecessary button Bless inventary

fix: old protocol hide unnecessary button Bless inventary

kokekanon committed:
fix: bug login quickly unfinished http post

fix: bug login quickly unfinished http post

rodrigopaixaorj committed:
Merge pull request #5 from kokekanon/new-layout

Feat: icons game_skills | improve: client_background | fix: bug login  unfinished http post and hide unnecessary button

rodrigopaixaorj committed:
Minor changes

* Replaced some states icons
* Added condition icons (kokekanon)
* Fixed cap and soul (kokekanon)

kokekanon committed:
Fix: border label text item

kokekanon committed:
Merge remote-tracking branch 'upstream/new-layout' into new-layout

kokekanon committed:
original presence

kokekanon committed:
fail try to fix tab key

kokekanon committed:
feat: Top Menu

kokekanon committed:
fix: accidentally deleted in dace42d

accidentally deleted
 dace42d
https://github.com/Nottinghster/otclient/commit/dace42d33daf0c522ac7bc7c548dd2bf347b8432

kokekanon committed:
delete local , posible use in top menu

kokekanon committed:
Create DELETE THIS.md

kokekanon committed:
Update DELETE THIS.md

rodrigopaixaorj committed:
Update setup.otml

rodrigopaixaorj committed:
Merge pull request #6 from kokekanon/new-layout

fix : Item label text, improve: topmenu, bottommenu , fix:bot , fix: enter_game tab

kokekanon committed:
fix: Opacity effect and missile in client_options 676

fix: Opacity effect and missile in client_options 676

kokekanon committed:
fix: icons of ( cooldown, vBot Analyzers, bot,  task)

creation of icons for:
- vbot Analizer
- bot
- cooldowns
- tasks

fix: break compatibility ex.client_topmenu.addRightGameToggleButton now game_mainpanel.addToggleButton

rodrigopaixaorj committed:
Update README.md

Commit:
update

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

kokekanon committed:
Update README.md

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

kokekanon committed:
fix: consoleTextEdit focus + fix: icons

Fix: ico of discord and terminal
Fix: consoleTextEdit focus
improve : "Customise Character" in game_interface for new protocol

kokekanon committed:
fix: toggle() topmenu incorrect anchor

kokekanon committed:
fix: display fps , ping tibia 13

kokekanon committed:
Update README.md

kokekanon committed:
fix: bug when attempting to open a container with no space in gameRightPanel.

kokekanon committed:
fix: Bug occurs when attempting to move a miniwindow to locations other than gameRightPanel, gameRightExtraPanel, gameLeftPanel, or gameLeftExtraPanel

kokekanon committed:
cleanup: orden getpanel of game_interface

kokekanon committed:
Improve: [BOT] check and open the left panel if it's closed when clicking on the bot button icon.

rodrigopaixaorj committed:
Merge pull request #8 from kokekanon/new-layout

fix: not displaying ping and fps.

kokekanon committed:
Merge branch 'main' into new-layout

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

kokekanon committed:
fix: missing image

kokekanon committed:
fix: label fps

kokekanon committed:
fix: Missing images from topmenu Ping (PR # 8)

kokekanon committed:
fix: width mini windows of bot

kokekanon committed:
Merge remote-tracking branch 'upstream/new-layout' into new-layout

rodrigopaixaorj committed:
Update README.md

kokekanon committed:
fix: FPS is showing even if you have disable showFPS in options (close and open client)

kokekanon committed:
Fix: chase combat control

kokekanon committed:
Merge remote-tracking branch 'upstream/new-layout' into new-layout

kokekanon committed:
fix: missing code of topmenu píng

kokekanon committed:
fix : onStatesChange top

kokekanon committed:
Update README.md

kokekanon committed:
fix: onStatesChange when StatsBar is hidden

rodrigopaixaorj committed:
improve: clipsoft icons

rodrigopaixaorj committed:
Merge branch 'new-layout' of https://github.com/Nottinghster/otclient into new-layout

rodrigopaixaorj committed:
fix: fix typo

rodrigopaixaorj committed:
improve: remove unnecessary spaces

kokekanon committed:
feat: add "manager Accounts" button to game_topmenu

kokekanon committed:
fix: onStatesChange when the inventory is minimized.

 The logic needs improvement.

kokekanon committed:
fix: hide "expert pvp modes" for old protocols

kokekanon committed:
Update README.md

rodrigopaixaorj committed:
remove: removed duplicated skills icons

rodrigopaixaorj committed:
Merge branch 'new-layout' into new-layout

rodrigopaixaorj committed:
Merge pull request #12 from kokekanon/new-layout

fix: Chase combat control

kokekanon committed:
Update README.md

kokekanon committed:
fix: game_topmenu try retrocompatibility

kokekanon committed:
fix: 'chaseModeRadioGroup' (a nil value) #12

kokekanon committed:
feat: UI/UX classic + lockButton

classic:
- Checkbox
-miniwindows buttons
- scroll bar

kokekanon committed:
feat: Cooldown spell on top of the game_console like tibia 13

kokekanon committed:
Update README.md

rodrigopaixaorj committed:
fix: icons location like clipsoft

rodrigopaixaorj committed:
Update README.md

rodrigopaixaorj committed:
Update README.md

kokekanon committed:
fix: offset icons state in statsbar

rodrigopaixaorj committed:
fix: problems with token label when no cache, (first open client)

rodrigopaixaorj committed:
Update README.md

kokekanon committed:
fix: Icon positioning in game_skills.

rodrigopaixaorj committed:
Update README.md

kokekanon committed:
fix: Minimized inventory panel doesn't show real values for Soul and Cap

kokekanon committed:
fix: game_cooldown if logout , the anchor does not exist

ERROR: cannot add anchors to widget 'cooldownWindow': the parent doesn't use anchors layout

rodrigopaixaorj committed:
fix: reorder buttons, like rl tibia

kokekanon committed:
fix: hide hotkeys Button

kokekanon committed:
feat: [Client_bottom] default information if array services is not enabled

kokekanon committed:
Update README.md

kokekanon committed:
feat: addStoreButton to mainpanel

Create long buttons of 108x20, one below the other

Commit:
update

Commit:
Merge remote-tracking branch 'upstream/main' into update-1332-luan

kokekanon committed:
fix: Height "special panel" caused in b52f153

fix: Height "special panel" caused in b52f153

kokekanon committed:
feat: add shopV8 compatible with attachEffect and shader

feat: add shopV8 compatible with attachEffect and shader

In the absence of another shop module, the one from v8 was added.

kokekanon committed:
Update README.md

rodrigopaixaorj committed:
Update README.md

rodrigopaixaorj committed:
fix: chase mode action in hotkeys

rodrigopaixaorj committed:
Update README.md

rodrigopaixaorj committed:
fix: fix typo

kokekanon committed:
fix: game_container like tibia 13

kokekanon committed:
fix: offset panel combobox of shader because collides with the ping and fps

kokekanon committed:
Update store_large.png

kokekanon committed:
Merge pull request #16 from luanluciano93/update-1332-luan

Update 1332 luan

kokekanon committed:
Update README.md

rodrigopaixaorj committed:
Update protocolgameparse.cpp

kokekanon committed:
temp fix: Web Scraping Topmenu

- limit the use of webscaping to php 82 versions only
- Replacement of Windows error with a print. Less invasive.

kokekanon committed:
fix: webscraping does not work on versions lower than php 82 #18

-add if (function_exists('curl_init')) {

rodrigopaixaorj committed:
remove: removed unsupported clients

kokekanon committed:
fix: main repo bug enter_game

kokekanon committed:
feat: setUniqueServer on init.lua like v8 request luanluciano93

request luanluciano93

kokekanon committed:
feat: repair client in option(if you find a problem in test)

feat: repair client in option(if you find a problem in test)

rodrigopaixaorj committed:
Merge pull request #20 from kokekanon/new-layout

feat: setUniqueServer on init.lua like v8 request luanluciano93

rodrigopaixaorj committed:
fix: Bug states onStatesChange bug

rodrigopaixaorj committed:
Update README.md

rodrigopaixaorj committed:
Minor changes

* Removed unnecessary spaces/tabs
* function inventoryController:onTerminate() now calls:
>> inventoryControllerEvents:disconnect()
>> inventoryControllerEvents_game:disconnect()

kokekanon committed:
fix: sort icons in addSpecialToggleButton

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

kokekanon committed:
feat: onChangeWorldTime | fix: bug mini map

check these functions on your server

old tfs: void ProtocolGame::sendWorldTime()

new tfs : function Player.sendWorldTime(self, time)

Canary: void ProtocolGame::sendTibiaTime(int32_t time)

kokekanon committed:
fix: Capacity and soul label when changing character.

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

kokekanon committed:
fix: Execute the last saved statusbar style correctly

kokekanon committed:
fix : [Game_bot ] get the slot5 requested by quiver_label and quiver_manager

luanluciano93 committed:
Update protocolgameparse.cpp

luanluciano93 committed:
Update protocolgameparse.cpp

luanluciano93 committed:
Update protocolgameparse.cpp

rodrigopaixaorj committed:
fix: correct version of blessings/expert pvp modes/customise characters

rodrigopaixaorj committed:
Update README.md

rodrigopaixaorj committed:
Update README.md

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

kokekanon committed:
Merge pull request #22 from luanluciano93/update-1332-luan

fix: protocol 13.32 by luan

luanluciano93 committed:
Update protocolgameparse.cpp

luanluciano93 committed:
Update protocolgameparse.cpp

luanluciano93 committed:
Update game.lua

kokekanon committed:
revert this, conflict 1332

kokekanon committed:
Merge pull request #23 from luanluciano93/update-1332-luan

Update 1332 luan

luanluciano93 committed:
Update protocolgameparse.cpp

kokekanon committed:
Merge pull request #24 from luanluciano93/update-1332-luan

feat: try 13.32

kokekanon committed:
feat: game_outfit v8

kokekanon committed:
Update README.md

kokekanon committed:
if someone makes a modification to the onOpenOutfitWindow

kokekanon committed:
fix: by default disable opcode

kokekanon committed:
add function to game_attachedeffects

- get category  by id
- get directory of image by id if category is 5
- get name of AttachedEffect by id

I don't improve the names because surely, a better way to do it will be found.

kokekanon committed:
fix: tabulation

kokekanon committed:
no feat : serverSIDE xd

kokekanon committed:
Update README.md

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

kokekanon committed:
fix: prevent error

kokekanon committed:
undo this , is for test

rodrigopaixaorj committed:
Merge pull request #25 from Nottinghster/featOutfitv8

feat: game_outfit v8 compatible with attachEffect , shader

rodrigopaixaorj committed:
Update options.lua

Removed duplicated line

kokekanon committed:
fix: missing content of feat: Typing Icon

luanluciano93 committed:
Update protocolgameparse.cpp

kokekanon committed:
Update README.md

rodrigopaixaorj committed:
Merge pull request #27 from luanluciano93/fix-parsePreyRerollPrice-1332

fix: parsePreyRerollPrice 13.32

kokekanon committed:
fix: enableLights client_options

kokekanon committed:
fix: order miniwindows Button

kokekanon committed:
fix: status bar (large mode) supports 6 icons

kokekanon committed:
Update README.md

Commit:
add hotkey button

^

kokekanon committed:
Review: kokekanon #28

rodrigopaixaorj committed:
Merge pull request #28 from DoktorHolzbein/new-layout

fix: add hotkey button

SkullzOTS committed:
Update 30-minimap.otui

kokekanon committed:
fix: #29 "Textmessage does not appear / does not show"

kokekanon committed:
Merge pull request #30 from SkullzOTS/patch-1

fix: #15

kokekanon committed:
Revert "fix: #15"

kokekanon committed:
Merge pull request #31 from SkullzOTS/patch-1

fix: #15  (correct branch)

kokekanon committed:
Merge pull request #32 from Nottinghster/revert-30-patch-1

Revert "fix: #15" (incorrect branch)

rodrigopaixaorj committed:
Update README.md

rodrigopaixaorj committed:
adjustments: minor changes

- Better offset icons state in statsbar
- Minor fix in minimap for bot

rodrigopaixaorj committed:
fix: skills icon invisible with percent bar

kokekanon committed:
feat: GameWingsAurasEffects connection server

rodrigopaixaorj committed:
Update attachedeffects.lua

rodrigopaixaorj committed:
Merge pull request #34 from Nottinghster/feat-outfit-server

feat: GameWingsAurasEffects connection server

rodrigopaixaorj committed:
Update README.md

rodrigopaixaorj committed:
Update README.md

kokekanon committed:
feat: game_outfit compatibility with shader

feat: game_outfit compatibility with shader

mehah committed:
Centralized UICreature

mehah committed:
fix uicreature size

mehah committed:
fix walk animation on stop static walking

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

rodrigopaixaorj committed:
Update README.md

kokekanon committed:
adjustments: [game_outfit] minor changes

luanluciano93 committed:
Update protocolgameparse.cpp

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
Merge pull request #37 from luanluciano93/ajust-1332

fix: getItem byte in 13.32

rodrigopaixaorj committed:
clean: remove old outfit window module

kokekanon committed:
fix: Decrease (Left , LeftExtra, RightExtra) with miniwindows inside

notthinghster: "Could you test one thing?

1 - Increase left or right panel, move any window (example: skills window) to the opened panel
2 - Decrease/close the panel, and then, try to open the skills window again, it remains in decreased panel

The expected scenario is:

If there are any window opened in left or right panel, and they are closed/decreased, they should return to main right panel
Know how to fix it ?
Rl tibia works like this"

rodrigopaixaorj committed:
clean: removed test codes from Kokekanon

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
improve: better logic in movepanel function

- Now it works like RL Tibia (the widget now closes only after confirming that there is a panel available to move it.)

rodrigopaixaorj committed:
fix: outfit size in right panel of outfit window

Thanks to @SkullzOTS

rodrigopaixaorj committed:
Revert "fix: outfit size in right panel of outfit window"

This reverts commit a45b9cc36da44dac6fdf1ab790b16ca500c93ceb.

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

kokekanon committed:
init

kokekanon committed:
Update mainpanel.lua

rodrigopaixaorj committed:
Merge pull request #42 from Nottinghster/fix-combat

fix: set safe fight option 2

rodrigopaixaorj committed:
fix: combat controls (pvp mode)

rodrigopaixaorj committed:
revert: combat controls (not fixed yet)

luanluciano93 committed:
Update entergame.lua

luanluciano93 committed:
Update entergame.lua

rodrigopaixaorj committed:
Merge pull request #44 from luanluciano93/fix-entergame

fix: error and ajust settings in entergame.lua

luanluciano93 committed:
update

luanluciano93 committed:
Merge branch 'new-layout' into fix-forge-convergence

mehah committed:
fix outfit and mount render size

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

rodrigopaixaorj committed:
Merge pull request #46 from luanluciano93/fix-forge-convergence

fix: bytes convergence forge 13.32

kojimhojo commented on README.md:
```diff
@@ -1,178 +1,292 @@
-- Floating Effects Option
-- Optimized Terminal
-- Refactored Walk System
-- Support for more mouse buttons, for example 4 and 5
```
readme  shouldn't replace the original.

kojimhojo commented on data/setup.otml:
```diff
@@ -29,7 +29,7 @@ game
 
   render
     invisible-ticks-per-frame: 75
-    item-ticks-per-frame: 500
```
What does this have to do with the new layout?

kojimhojo commented on modules/corelib/http.lua:
```diff
@@ -181,8 +181,7 @@ function HTTP.onPost(operationId, url, err, data)
     end
-    -- data = result
-
+    data = result
```
Where is this used?

kojimhojo commented on modules/gamelib/game.lua:
```diff
@@ -58,7 +58,7 @@ function g_game.getSupportedClients()
         1051, 1052, 1053, 1054, 1055, 1056, 1057, 1058, 1059, 1060, 1061, 1062, 1063, 1064, 1070, 1071, 1072, 1073,
         1074, 1075, 1076, 1080, 1081, 1082, 1090, 1091, 1092, 1093, 1094, 1095, 1096, 1097, 1098, 1099, 1100, 1281, 1285,
-        1286, 1287, 1291, 1300, 1314, 1316, 1320, 1321 }
+        1286, 1287, 1291, 1300, 1310, 1311, 1314, 1316, 1320, 1321, 1322, 1332 }
```
Are you sure that 1310 works on this client?

kojimhojo commented:
What are the bugs // what are the missing features?

I think it would be a good idea to split

game mainpanel
in
- game minimap
- game_inventory
- game_mainpanel (healthMana + options)

so as not to confuse

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj commented:
> What are the bugs // what are the missing features?
> 
> I think it would be a good idea to split
> 
> game mainpanel in
> 
>     * game minimap
> 
>     * game_inventory
> 
>     * game_mainpanel (healthMana + options)
> 
> 
> so as not to confuse

Be free to help, you can do a commit

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

rodrigopaixaorj committed:
fix compile error

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
minor changes

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

rodrigopaixaorj committed:
Update game.lua

- Added again version 13.10/13.11, due to compatibility with main TFS

mehah committed:
Merge branch 'main' into new-layout

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
fix: minimap .white cross out of bounds

revert commit [ccb6e3f](https://github.com/Nottinghster/otclient/commit/ccb6e3f993ecbf99deb9b67566b234edd34492ce)

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
Bot server fixes

- modified the maxreconnectattempts
- added capped Attempts so delay will never be over 5 attempts
- added on error looking for resolve error to stop bot connecting
Fixes game edit windows not triggering.
- g_lua.callGlobalField("g_game", "onEditText", id, itemId, maxLength, text, writer, date);

Credits: https://github.com/Leesneaks/otclientv8/

kojimhojo commented:
@Nottinghster There’s an error in the tooltips; when an object is destroyed or hidden, the tooltip remains stuck with the mouse

rodrigopaixaorj commented:
> @Nottinghster There’s an error in the tooltips; when an object is destroyed or hidden, the tooltip remains stuck with the mouse

It's a known error by me, it's in my to-do list to fix ;)

kojimhojo commented:
@kokekanon you can help me ?

kokekanon commented:
> @kokekanon you can help me ?

![2414vv](https://github.com/mehah/otclient/assets/114332266/11926eeb-bf98-404c-b861-c76ec91d4719)
in
https://github.com/Nottinghster/otclient/blob/new-layout/modules/corelib/ui/tooltip.lua#L62
add
```lua
local function onWidgetDestroy(widget)
    if widget == currentHoveredWidget then
        if widget.tooltip then
            g_tooltip.hide()
        end
        if widget.specialtooltip then
            g_tooltip.hideSpecial()
        end
        currentHoveredWidget = nil
    end
end

```

add 
`onDestroy = onWidgetDestroy,`
in
 https://github.com/Nottinghster/otclient/blob/new-layout/modules/corelib/ui/tooltip.lua#L119
https://github.com/Nottinghster/otclient/blob/new-layout/modules/corelib/ui/tooltip.lua#L148

rodrigopaixaorj committed:
improve: better warning message

Based in Discord channel -> https://discord.com/channels/888062548082061433/888067068207960094/1241087185252388935

kojimhojo commented:
Obrigado

rodrigopaixaorj committed:
fix: when an object is destroyed or hidden, the tooltip remains stuck with the mouse

Credits: @kokekanon

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

rodrigopaixaorj committed:
minor fixes

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

mehah committed:
fix options

mehah committed:
Update blank.png

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
Merge remote-tracking branch 'upstream/main' into new-layout

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

kojimhojo commented:
![image](https://github.com/mehah/otclient/assets/164669689/ed168a45-1180-4309-b124-8c9d361716ef)

this does not work

rodrigopaixaorj commented:
> ![image](https://private-user-images.githubusercontent.com/164669689/338275063-ed168a45-1180-4309-b124-8c9d361716ef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgwNDI3NTgsIm5iZiI6MTcxODA0MjQ1OCwicGF0aCI6Ii8xNjQ2Njk2ODkvMzM4Mjc1MDYzLWVkMTY4YTQ1LTExODAtNDMwOS1iMTI0LThjOWQzNjE3MTZlZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxMFQxODAwNThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lYzEzZTBhYjAwZmE1OTExNzFlYTdmOWQ2ZTgwYmY5YmQ3OGRhNDExMGQ2MTA5YTNmMzg0NDMzYzFhMGMyNTViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.I_5LKOMK8XLP_NMAia1CoTrEhbXvaDAtk8txBYyHnpk)
> 
> this does not work

It's a known issue, but right now, don't have time to fix it
You can help us with this fix

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

rodrigopaixaorj committed:
Merge branch 'mehah:main' into new-layout

majestyotbr committed:
readme
